django/contrib/admin/templates/admin/includes/edit_actions.html

```diff
@@ -0,0 +1,7 @@
+{% if actions %}
+<div class="actions">
+    {% for action in actions.values %}
+    <input type="submit" name="{{action.1}}" value="{{action.2}}">
+    {% endfor %}
+</div>
+{% endif %}
```

Comment:
First off: love this PR, it's very much a desire-able feature for us (@yunojuno) and we have been close to implementing it for ourselves a few times now. We keep admin inline actions for the kind of day-to-day support tasks that only developers should carry out and running them currently is a bit of a hassle.

I see the UI here though the problematic bit, and probably the reason this will meet resistance. On certain models we have say 10 inline actions - and with the UI as it is that would print 10 buttons to the screen. We'd be inundated with buttons.

A potential alternative is to go with the same UI as the listing pages. i.e A `select` based dropdown with a single button to carry out the action.

Reply:
> A potential alternative is to go with the same UI as the listing pages

Finally applied!